src/pages/CommPartnersPage.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+
+import CommPartnersIcon from '../assets/community-partners.js';
+import MailIcon from '../assets/email-icon.js';
+import TechDesign from '../assets/tech-design.js';
+
+import '../styling/CommPartnersPage.css';
+
+const outreachPartners = [
+  { name: 'HackBeanpot', url: 'https://hackbeanpot.com/' },
+  { name: 'Khoury College of Computer Sciences', url: 'https://www.khoury.northeastern.edu/' },
+  { name: 'Unofficial Committee', url: 'https://www.unofficialcommittee.com/' },
+  { name: 'Common Impact', url: 'https://commonimpact.org/' }
+];
+
+const servicePartners = [
+  { name: 'Unofficial Committee', url: 'https://www.unofficialcommittee.com/' },
+  { name: 'Support Community, Inc', url: 'https://supportcommunity.com/' },
+  { name: 'Community Design Studio, Lesley Art + Design', url: 'http://lesley.smartcatalogiq.com/en/2020-2021/Undergraduate-Academic-Catalog/Undergraduate-Programs-and-Courses/College-of-Art-and-Design-Courses/IDESN-Design/3000/IDESN-3370' },
+];
+
+const CommPartnersPage = () => (
+  <Container className="comm-partners-page" fluid>
+    <Row>
+      <h1>Community Partners</h1>
+    </Row>
+    <Row className="comm-align-center">
+      <Col xs={12} md={6}>
+        <CommPartnersIcon className="mx-auto d-block comm-icon" fluid />
+      </Col>
+      <Col>
+        <p>
```

Comment:
wondering how this would look centered or in a larger font? right now the beginning paragraph seems lacking (since we cut it down) but contains important info to draw attention to.